Emby.Server.Implementations/Playlists/PlaylistManager.cs

```diff
@@ -56,10 +56,8 @@ public async Task<PlaylistCreationResult> CreatePlaylist(PlaylistCreationRequest
         {
             var name = options.Name;
 
-            var folderName = _fileSystem.GetValidFilename(name) + " [playlist]";
-
+            var folderName = _fileSystem.GetValidFilename(name);
```

Comment:
Why was this removed?

Reply:
This was just a style change. All internal playlists are stored in a folder called `playlists` so the additional string is redundant.